deploy/csi-azurelustre-node.yaml

```diff
@@ -132,6 +132,16 @@ spec:
               name: azurelustre-cache
             - mountPath: /dev
               name: host-dev
+            - mountPath: /host/var/log
```

Comment:
why would you need `/var/log` hostPath mount?

Reply:
That is where logs for Lustre utils go.